packages/design-system/src/atoms/popover/index.web.tsx

```diff
@@ -73,27 +74,29 @@ export const Popover = ({
         open,
       })}
       {open && (
-        <View
-          style={[
-            container.borderRadiusSM,
-            container.shadow,
-            // @ts-ignore
-            // eslint-disable-next-line react-native/no-inline-styles
-            {
-              zIndex: 999,
-              top: y ?? 0,
-              left: x ?? 0,
-              backgroundColor: theme.background.primary,
-              position: strategy,
-            },
-            style,
-          ]}
-          ref={refs.setFloating}
-          {...getFloatingProps()}
-          {...rest}
-        >
-          {children}
-        </View>
+        <FloatingPortal>
+          <View
+            style={[
+              container.borderRadiusSM,
+              container.shadow,
+              // @ts-ignore
+              // eslint-disable-next-line react-native/no-inline-styles
+              {
+                zIndex: layer.layer999,
+                top: y ?? 0,
```

Comment:
Is nullish coalescing (??) used here to return 0 as a default value?

Reply:
yep :)

Reply:
I see, thanks :)